### PR TITLE
Change all SQL keywords to uppercase

### PR DESCRIPTION
**Pull request by ream88**
I know this may be ridiculous.

UPCASE ALL THE KEYWORDS

### CODE REVIEW

File: lib/queue_classic/queue.rb [hunk 1]
```diff
@@ -89,14 +89,14 @@ def lock
 
     def unlock(id)
       QC.log_yield(:measure => 'queue.unlock') do
-        s = "UPDATE #{QC.table_name} set locked_at = null where id = $1"
+        s = "UPDATE #{QC.table_name} SET locked_at = null WHERE id = $1"
```
Comment (smathieu) [inline lib/queue_classic/queue.rb]: null => NULL

**Comment by ream88** [comment]
Changed the `null` and rebased with master

**Comment by ukd1** [comment]
👍 thx!

**Comment by ream88** [comment]
ty for merging :)

**Pull request merged**